samples/snippets/snippets_notification_configs.py

```diff
@@ -124,19 +124,21 @@ def update_notification_config(organization_id, notification_config_id, pubsub_t
 
     # Only description and pubsub_topic can be updated.
     field_mask = field_mask_pb2.FieldMask(
-        paths=["description", "pubsub_topic", "streaming_config.fitler"]
```

Comment:
Was this failing because this was a typo? `fitler` instead of `filter`?

Reply:
Oh Nice catch, I'll update :)